src/entity.cpp

```diff
@@ -3,6 +3,7 @@
 int currentId = 0;
```

Comment:
Its for incrementing the entity id. It cannot be a private variable because once it will always be a new variable with each new entity.